cmd/operator-sdk/run/local.go

```diff
@@ -52,9 +53,15 @@ type runLocalArgs struct {
 func (c *runLocalArgs) addToFlags(fs *pflag.FlagSet) {
 	prefix := "[local only] "
 
-	fs.StringVar(&c.watchNamespace, "watch-namespace", "",
-		prefix+"The namespace where the operator watches for changes. Set \"\" for AllNamespaces, "+
-			"set \"ns1,ns2\" for MultiNamespace")
+	// The main.go and manager.yaml scaffolds in the new layout do not support the WATCH_NAMESPACE
+	// env var to configure the namespace that the operator watches. The default is all namespaces.
+	// So this flag is unsupported for the new layout.
+	if !kbutil.HasProjectFile() {
+		fs.StringVar(&c.watchNamespace, "watch-namespace", "",
+			prefix+"The namespace where the operator watches for changes. Set \"\" for AllNamespaces, "+
+				"set \"ns1,ns2\" for MultiNamespace")
```

Comment:
For the first contidional [here](https://github.com/operator-framework/operator-sdk/blob/48b720395368cec82a9e2f2530d3d460557bb216/cmd/operator-sdk/run/cmd.go#L98-L104).
It will set watch-namepace == namespace if the flag is set. This flag does not exist when it is in the new layout. 

However, for the second conditional [here](https://github.com/operator-framework/operator-sdk/blob/48b720395368cec82a9e2f2530d3d460557bb216/cmd/operator-sdk/run/cmd.go#L105-L112) I think would be better set to be cluster-scoped and you are right. I will do that.